content/hi/docs/reference/glossary/api-eviction.md

```diff
@@ -0,0 +1,26 @@
+---
+title: API-
```

Comment:
`title: API-` may require correction as it doesn't align with the corresponding English [file](https://github.com/shelar1423/website/blob/main/content/en/docs/reference/glossary/api-eviction.md?plain=1).